src/components/common/typography/Typography.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useTheme } from '@chakra-ui/react';
+
+interface TypographyProps {
+  children: React.ReactNode;
+  variant?: 'h1' | 'h2' | 'h3' | 'p';
+  color?: string;
+  fontSize?: string;
+  fontWeight?: string;
+  marginBottom?: string;
+  className?: string;
+  style?: React.CSSProperties;
+}
+
+const Typography: React.FC<TypographyProps> = ({
```

Comment:
The Typography component is not a single component; it should include H1, H2, ..., H6, etc.